tools/replace_prairielib_paths.py

```diff
@@ -40,22 +40,22 @@ def walk_directory(dirname, old_string, new_prefix, new_string=None):
 walk_directory('.', "require('@prairielearn/prairielib').error", "require('@prairielearn/prairielib/error')")
```

Comment:
We should just remove this whole Python file entirely, it's not useful anymore.

Reply:
done